x-pack/legacy/plugins/apm/cypress/README.md

```diff
@@ -0,0 +1,57 @@
+### How to run
+
+_Note: Run the following commands from `kibana/x-pack/legacy/plugins/apm/cypress`._
```

Comment:
Seems like the tests requires that we start kibana with the `--no-base-path` option, probably should either fix that or document it here.